x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_alert_list.tsx

```diff
@@ -91,6 +91,21 @@ const alertsTableColumns = (
     width: '80px',
     'data-test-subj': 'alertsTableCell-duration',
   },
+  {
+    field: 'maintenanceWindowIds',
+    width: '250px',
+    render: (value: string[]) => {
+      return Array.isArray(value) ? JSON.stringify(value) : '';
+    },
+    name: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.alertsList.columns.maintenanceWindowIds',
+      {
+        defaultMessage: 'Maintenance window IDs',
```

Comment:
IMO the "IDs" is unnecessary here:
```suggestion
        defaultMessage: 'Maintenance windows',
```